src/utils/taskOverDueDiscordMembers.ts

```diff
@@ -1,24 +1,20 @@
 import { RDS_BASE_API_URL } from "../constants/urls";
-import {
-  TaskOverdue,
-  TaskOverdueResponse,
-} from "../typeDefinitions/taskOverdue.types";
-import * as errors from "../constants/responses";
+import { UserOverdueTaskResponseType } from "../typeDefinitions/rdsUser";
 
 export const taskOverDueDiscordMembers = async (): Promise<
   string[] | string
 > => {
   try {
-    const overDueUrl = `${RDS_BASE_API_URL}/tasks?dev=true&status=overdue&size=100`;
+    const overDueUrl = `${RDS_BASE_API_URL}/users?query=filterBy:overdue_tasks`;
```

Comment:
Are you sure this API will return tasks of only those users in the discord server?

Reply:
Yes